src/cache/precomputed_quantities.jl

```diff
@@ -170,6 +170,15 @@ function precomputed_quantities(Y, atmos)
         ᶜgradᵥ_θ_liq_ice = Fields.Field(C3{FT}, cspace),
     )
 
+    diagnostic_edmf_precipitation_quantities =
```

Comment:
```suggestion
    diagnostic_precipitation_sgs_quantities =
```
(to be consistent with the other names)